pkgs/development/python-modules/pytest/default.nix

```diff
@@ -23,6 +23,8 @@ let generic = { version, sha256 }:
       ++ stdenv.lib.optionals (!isPy3k) [ funcsigs ]
       ++ stdenv.lib.optionals (pythonOlder "3.6") [ pathlib2 ];
 
+    # https://hydra.nixos.org/build/87478471/nixlog/1
+    doCheck = false;
```

Comment:
That's for `pytest_32` used by `celery`. This has been fixed in fae1f7afc144ba11bc8fa340878b85956291280b, which will be merged with https://github.com/NixOS/nixpkgs/pull/54306.